### PR TITLE
added Slideshow

### LINKED ISSUE #519: Write "Developer Advocate/ Evangelist" topic outline
The intention is to present Developer Advocate/ Evangelist roles, and steps to take if one is interested in a career in That field.  

- [x] Create a topic outline [here](https://github.com/Techtonica/curriculum/blob/master/roles-in-tech/developer-advocate.md).

- [ ] make a slideshow version of the topic. You can make a copy of this old Dev Evangelist slideshow and customize: https://docs.google.com/presentation/d/15Ws1i7szpyhzHhLuHa_LTo9uNGrHg4bujcHq0we2fGM/edit?usp=sharing

Acceptance Criteria:
- [x] The lesson can be covered in 30-45 minutes
- [x] There are clear objectives
- [x] The activity helps different kinds of learners (e.g. spatial, auditory, verbal, kinesthetic)
- [x] There is a question-based assessment for Check for Understanding

Note that [self and peer assessments are preferable to grades](https://www.mindsetkit.org/topics/assessments-growth-mindset-math/using-self-peer-feedback-assessments-learning).

### PR DESCRIPTION
**Pull request by yati1998**
fixes #519

### CODE REVIEW

File: roles-in-tech/developer-advocate.md [hunk 1]
```diff
@@ -2,89 +2,174 @@
...
-### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
 
 ### Motivation
 
-Here's why this topic should be learned. Strengthen this section by including a company that uses this skillset, and a screenshot example if possible.
+A developer evangelist is a spokesperson, mediator and translator between a company and both its technical staff and outside developers. It's not always about writing codes. What if you don't write codes?
```
Comment (manaswinidas) [inline roles-in-tech/developer-advocate.md]: code in place of codes

File: roles-in-tech/developer-advocate.md [hunk 2]
```diff
@@ -2,89 +2,174 @@
...
+You can still set your position in your dream companies like Google. All the top companies or you may say brands have a position of "Developer Advocate". And on following the correct steps you can be one of them.
 
-### Objectives
 
-**Participants will be able to:**
+### Objectives
+The main objectives of this lesson are mentioned below:
+- To acknowledge the people about advantages of being a Developer advocate.
+- Basic skills required for being in that field.
+- Enhance those skills with proper information and guidence.
```
Comment (manaswinidas) [inline roles-in-tech/developer-advocate.md]: It's guidance

File: roles-in-tech/developer-advocate.md [hunk 3]
```diff
@@ -2,89 +2,174 @@
...
-
-### Common Mistakes / Misconceptions
-
-List things that apprentices might not realize, might assume at first, or should avoid.
+### Materials
 
-- Example
-- Example
+Link to the SlideShare:
+https://docs.google.com/presentation/d/1JF2t4VCznUzD2tyW9kuc0Gx8-q0rnpzjDpXxpMFh8DA/edit#slide=id.p
```
Comment (manaswinidas) [inline roles-in-tech/developer-advocate.md]: Use markdown for links

File: roles-in-tech/developer-advocate.md [hunk 4]
```diff
@@ -2,89 +2,174 @@
...
+- https://talkpython.fm/episodes/show/189/war-stories-of-the-developer-evangelists
+- http://www.devangel.io/
+- https://devrelweekly.com/issues
 
 
 ### Guided Practice
 
-Have the apprentices work with you as you do something step-by-step.  This can also be fulfilled by a detailed tutorial intended for beginners.
+Here is a link to the complete tutorial for the developer advocate beginners:
+http://developer-evangelism.com/handbook.php
```
Comment (manaswinidas) [inline roles-in-tech/developer-advocate.md]: Please use markdown for links

File: roles-in-tech/developer-advocate.md [hunk 5]
```diff
@@ -2,89 +2,174 @@
...
 
 
 ### Independent Practice
 
-Class does this thing themselves with specific additional items. This could be alone, with a partner, or small group; but the idea is that it's less guided, more independent.
+Further after completing the lessons in the guided practice one must do the following things to enhance there skills:
```
Comment (manaswinidas) [inline roles-in-tech/developer-advocate.md]: their in place of there

File: roles-in-tech/developer-advocate.md [hunk 6]
```diff
@@ -2,89 +2,174 @@
...
 
 ### Challenge
 
-Apprentices can try to do this other thing. Ideally, they will be challenged to connect what they've learned to some previous knowledge or additional research.
-
-
-### Check for Understanding
+Make a group of 4-5 .
+Perform the Group Disscussion among yourself. Group Disscussion can be on any topic. It will enhance your speaking skills.
+Practice soft skills by regular interraction with different people.
```
Comment (manaswinidas) [inline roles-in-tech/developer-advocate.md]: It's interaction

**Comment by alodahl** [comment]
@yati1998 , if a slideshow is part of your PR, please leave a link in the description!

**Comment by alodahl** [comment]
@yati1998 , can you please put more images in the slideshow and create a more bulleted format.  The goal is to break up the big blocks of text that are hard to read - the slides are intended for visual learners.

**Comment by yati1998** [comment]
@alodahl  and @manaswinidas I have made the required changes.

File: roles-in-tech/developer-advocate.md [hunk 7]
```diff
@@ -149,14 +152,17 @@ Here is a link to different articles which will let you know much about a Develo
 - ["Reference8"](https://devrelweekly.com/issues)
```
Comment (alodahl) [inline roles-in-tech/developer-advocate.md]: Please title reference links with title of reference page, and the kind of reference: article, blog, video, etc

Comment (yati1998) [reply roles-in-tech/developer-advocate.md]: ohk sure. I will be doing that.

File: roles-in-tech/developer-advocate.md [hunk 8]
```diff
@@ -10,16 +10,17 @@
 
 
 ### Motivation
+A developer evangelist is a spokesperson, mediator and translator between a company and both its technical staff and outside developers. It's not always about writing code. What if you don't write codes?
```
Comment (alodahl) [inline roles-in-tech/developer-advocate.md]: codes? -> code?

**Comment by yati1998** [comment]
@alodahl I have made the required changes in the slide as well as outline. Please have a look to it.

**Pull request merged**